### ISSUE TITLE
could support the Dropout layer?

### ISSUE THREAD

**Issue by xinsuinizhuan** [jatinchowdhury18/RTNeural#62]
I want use it for forecast, and custome the struct like:
model = Sequential()
model.add(LSTM(units=50, return_sequences=True, input_shape=(forecast_features_set.shape[1], 1)))
model.add(Dropout(0.2))
model.add(LSTM(units=50, return_sequences=True))
model.add(Dropout(0.2))
model.add(LSTM(units=50, return_sequences=True))
model.add(Dropout(0.2))
model.add(LSTM(units=50))
model.add(Dropout(0.2))
model.add(Dense(units = 1))
model.compile(optimizer = 'adam', loss = 'mean_squared_error')

**Comment by jatinchowdhury18** [comment]
At the moment RTNeural only supports running inference on neural networks models that have already been trained. To my knowledge Dropout layers are typically only used during the training process, and not during inference as well.
